Userland/Libraries/LibPDF/Renderer.h

```diff
@@ -93,6 +93,7 @@ class Renderer {
     void handle_text_next_line_show_string_set_spacing(const Vector<Value>& args);
 
     // shift is the manual advance given in the TJ command array
```

Comment:
This comment seems to refer to the `show_text` function, but is now refering to `set_graphics_state_from_dict`.

Reply:
Nice catch! Would have been confusing for anyone coming across that comment :)